students/GuyCChan/session04/trigrams.py

```diff
@@ -0,0 +1,82 @@
+import string
+import random
+
+def read_in_data():
+# Open text file and put file body in a list.
+    with open('./sherlock_small.txt', 'rb') as file:
+        file_body = file.readlines()
+        allwords = []
+        for line in file_body:
+            line = line.strip()
+            if line.endswith("in part my own."):
+                break
+            allwords.append(line)
```

Comment:
allwords is now a list of strings, with each string a line in the text.

is that what you expect?